CHANGELOG.md

```diff
@@ -14,6 +15,7 @@
 
 **Details**
 - `nflow-engine`
+  - Add `priority` two byte integer to the `nflow_workflow` table. When an executor chooses from many available scheduled workflow instances it primarily (unfairly) schedules the workflow instance with the larger priority value, and for workflows with the same priority, the one scheduled first. Priority defaults to 0 and can also be negative. Requires database migration.
```

Comment:
Maybe mention here that priority can be set on definition (default) and instance (override) level?

Reply:
Or should I update this in my pr...

Reply:
yeah, thought that this would be a two PR thing, as you already did your own :)